server/app/app.py

```diff
@@ -118,6 +118,17 @@ def dataroot_index():
     return make_response(data)
 
 
+def dataroot_index():
+    # Handle the base url for the cellxgene server when running in multi dataset mode
+    config = current_app.app_config
+    if config.multi_dataset_index is None:
```

Comment:
None or False for safety?   or `if not config.multi_dataset_index: ...`

Reply:
Done